fdbclient/NativeAPI.actor.cpp

```diff
@@ -904,8 +919,8 @@ void setNetworkOption(FDBNetworkOptions::Option option, Optional<StringRef> valu
 				fprintf(stderr, "FoundationDB client ignoring unrecognized knob option '%s'\n", knobName.c_str());
 			}
 			break;
-		}
-		case FDBNetworkOptions::TLS_PLUGIN:
+	    }
+	    case FDBNetworkOptions::TLS_PLUGIN:
```

Comment:
Something weird has happened with tabs and spaces between 890 and here.

Reply:
Somehow the Clang-format commit hook decides to change some tabs into spaces.

Reply:
Done